### DISCUSSION TITLE
connect docker to azure market place through microsoft account

### DISCUSSION THREAD

**Discussion by IBOCATA** [docker/extension-ideas#185] [Ideas]
Very helpful for azure container registry

**Comment by benja-M-1** [comment]
Do you have a documentation on how to use docker and the Azure Marketplace? I don't see how the two would work together

**Reply by IBOCATA** [reply]
Hello Benjamin,
This is the documentation.
https://microsoft.github.io/Mastering-the-Marketplace/partner-center/pc-create-offers/

**Comment by smhwangd73** [comment]
[CONTRIBUTING.md](https://github.com/user-attachments/files/16145547/CONTRIBUTING.md)
